course/views/views.py

```diff
@@ -69,9 +58,6 @@ def question_view(request, pk, key=None):
     if isinstance(question, JavaQuestion):
         return _java_question_view(request, question, key)
 
-    if isinstance(question, CheckboxQuestion):
-        return _multiple_choice_question_view(request, question, 'checkbox_question.html', key)
```

Comment:
The html files that are not being used anymore should also be deleted.

Reply:
Do I delete just the html files or do I delete all the rendering related to problems?